ios/RNCNetInfo.m

```diff
@@ -125,8 +129,14 @@ - (NSMutableDictionary *)detailsFromInterface:(nonnull NSString *)requestedInter
     details[@"ipAddress"] = [self ipAddress] ?: NSNull.null;
     details[@"subnet"] = [self subnet] ?: NSNull.null;
     #if !TARGET_OS_TV && !TARGET_OS_OSX
-    details[@"ssid"] = [self ssid] ?: NSNull.null;
-    details[@"bssid"] = [self bssid] ?: NSNull.null;
+      // Without location permissions enabled, CNCopyCurrentNetworkInfo used to get these details will leak
+      if (CLLocationManager.locationServicesEnabled) {
+          CLAuthorizationStatus status = CLLocationManager.authorizationStatus;
+          if(status == kCLAuthorizationStatusAuthorizedAlways || status == kCLAuthorizationStatusAuthorizedWhenInUse) {
```

Comment:
Just a spacing nitpick, doesn't take away from what looks like a good fix based on the also-good writeup

```suggestion
          if (status == kCLAuthorizationStatusAuthorizedAlways || status == kCLAuthorizationStatusAuthorizedWhenInUse) {
```

Reply:
At least it wasn't a log 😄